app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -68,6 +68,9 @@ class DefaultToolbarMenu(
             menuItems,
             endOfMenuAlwaysVisible = !shouldReverseItems,
             store = store,
+            onAddonsManagerTapped = {
```

Comment:
nit: I think we need to provide a value for  `webExtIconTintColorResource`  without it, the icons don't show properly  
![image](https://user-images.githubusercontent.com/773158/82922706-f29b9b80-9f47-11ea-8598-aff1a5b3baaa.png)

Reply:
We could use `primaryTextColor()` :)

Reply:
Oops, added.